content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,91 @@
+
+### Authentication Process to Obtain the Access Token
+
+The DSCC public API relies on an OAuth 2.0 third party authorization framework on behalf of the resource owner (HPE GreenLake Console's user) for security. The user starts by logging and authenticating into HPE GreenLake Console, which is authenticated by the Identity Provider (validated through username, password, or MFA). Using the API Gateway menu in GreenLake Console, customer registers their client application to obtain the OAuth 2.0 API Client Credentials (Client ID and Client Secret). This association allows user to obtain the access-token from the menu, and user can then use the access-token into any REST API call inside the token bearer field. This action allows any client application or script to perform any HTTPS method to 
+
+The Access Token have a limited lifetime (about 720 seconds or 2 hours). Once it expires, the client application must use the obtained Client ID and Client Secret to obtain a new Access Token. One indication of the expiration of Access Token, the API will return response Error: '401 Unauthorized HTTP.'  If the user generate new Access Token prior to it expires, then the current Access Token will be invalidated or treated as not Authorized. 
+
+Additionally, user can also change the Client Secret to update the authorization when authorized user lost the Client Secret or the secret is compromised. And lastly, when access to the API must be disabled, user can delete the application association in the API Gateway.
+
+![Access API process](/img/user-guide-for-authorization.png "Process to authenticate and to obtain secure access ")
+
+## Accessing the API Gateway Console
+
+To access the API console, user must login into the [GreenLake Common Cloud](https:\common.cloud.hpe.com), deployed the Data Services Common Cloud Console Application, and onboarded a Storage Array (Alletra, Nimble and Primera) into the organization that is associated with the user account. The user must have the role that is required to perform the intended operation at the instance of DSCC where the storage is deployed. e.g. User must have volume management capability in the Data Ops Management to create a storage volume in US West. 
+
+The **API** **Gateway** menu is available inside **Manage** menu. From **Cloud Console** click on **Menu** to access this **Manage** Menu.
+
+![Access Menu ](/img/accesing-manage-menu-from-console.png "Menu in Cloud Console")
+
+![CCS Menu](/img/accesing-api-gateway-from-manage-menu.png "GreenLake Common Cloud Menu")
+
+The API Gateway provides the following operations:
+
+1. Create and manage API client applications association to obtain:
```

Comment:
recommend modifying to "Create and manage API client applications association in order to obtain:"